### PR TITLE
Be more PC

### PR DESCRIPTION
**Pull request by JanisGailis**
s/slave/replica

### CODE REVIEW

**Comment by JanisGailis** [comment]
@forman Will merge in a day, if you haven't reviewed :)

**Comment by JanisGailis** [comment]
Corresponding PR in ``cate-e2e``:
https://github.com/CCI-Tools/cate-e2e/pull/1

In other news:
Tests fail. Looks like deprecated xarray API that should be easy to fix.

**Comment by JanisGailis** [comment]
Looks like #795 must be merged

File: CHANGES.md [hunk 1]
```diff
@@ -7,6 +7,8 @@
 * Pearson correlation scalar operation now works on N-D data variables [#746](https://github.com/CCI-Tools/cate/issues/746)
 * Make sure integer data variables can be coregistered [#770](https://github.com/CCI-Tools/cate/issues/770)
 * Fix an issue with coregistering datasets with inverted latitudes [#733](https://github.com/CCI-Tools/cate/issues/733)
+* Fix deprecated API use to be able to use newer xarray versions
```
[the remark this answers is not in the document]
Comment (JanisGailis) [reply CHANGES.md]: Yes, I'll pull the new master and update the CHANGES.md accordingly

[the remark this answers is not in the document]
Comment (forman) [reply CHANGES.md]: Helge will look into #795 tomorrow, there is fiona 1.8.0 now.

[the remark this answers is not in the document]
Comment (JanisGailis) [reply CHANGES.md]: Great! I'd prefer if #795 is merged first.

**Pull request closed without merging**